src/main/java/us/kilroyrobotics/RobotContainer.java

```diff
@@ -343,14 +258,51 @@ private Command alignReef(boolean leftSide) {
                 });
     }
 
-    private Timer autoLeaveTimer = new Timer();
-    private Command autoLeave =
+    /* AutoLeave Command */
+    private final Timer autoLeaveTimer = new Timer();
+    private final Command autoLeave =
             Commands.sequence(
-                            Commands.runOnce(() -> this.autoLeaveTimer.restart()),
+                            Commands.runOnce(() -> autoLeaveTimer.restart()),
                             drivetrain.applyRequest(
                                     () -> forwardStraight.withVelocityX(MetersPerSecond.of(0.5))))
                     .onlyWhile(() -> !autoLeaveTimer.hasElapsed(2));
 
+    /* Path follower */
+    private final SendableChooser<Command> autoChooser;
+
+    @SuppressWarnings("unused")
+    public RobotContainer() {
+        if (Robot.isReal() && CameraConstants.kCameraEnabled) new Camera();
+
+        NamedCommands.registerCommand(
+                "Intake Coral",
+                towerIntakeCoral
+                        .andThen(Commands.waitTime(Seconds.of(3)))
+                        .until(() -> coralIntakeMotor.isCoralDetected()));
+        NamedCommands.registerCommand("Score Coral", towerScoreCoral);
+        NamedCommands.registerCommand(
+                "GoTo L1", towerToL1.andThen(Commands.waitTime(Seconds.of(2))));
+        NamedCommands.registerCommand(
+                "GoTo L2", towerToL2.andThen(Commands.waitTime(Seconds.of(2))));
+        NamedCommands.registerCommand(
+                "GoTo L3", towerToL3.andThen(Commands.waitTime(Seconds.of(2))));
+        NamedCommands.registerCommand(
+                "GoTo L4", towerToL4.andThen(Commands.waitTime(Seconds.of(2))));
+
+        NamedCommands.registerCommand("Leave", autoLeave);
+
+        autoChooser = AutoBuilder.buildAutoChooser();
+        SmartDashboard.putData("Auto Mode", autoChooser);
+
+        autoChooser.onChange(
+                (Command command) -> {
+                    if (command != null)
+                        System.out.println("[AUTO] Selected Route " + command.getName());
```

Comment:
Remove prints

Reply:
I would rather not, since this helps to make sure what we put in shuffleboard actually carries over, and can be used for debugging since we don't have akit.

Reply:
Shuffleboard is used instead of prints.

If you want logging, use epilogue or the various other logging solutions that don't require a full framework and code base overhaul

Reply:
Yeah but I want this to be easily verifiable to the drive team by having it show right there that it selected the correct autonomous.

Reply:
Did you know you can display the selected auto path on elastic? Like, the whole path like you can see in ascope but on elastic.

Probably not gonna implement that, but good to know for next year

Reply:
Too bad, see #43. Also works on Shuffleboard, but I might switch it to elastic tonight.